File: Hybrid_Task/Java/StringPalindrome.java
public class StringPalindrome {
public static void main(String[] args) {

String word = "madaM".toLowerCase();


String reverse = " ";

for(int index = 0; index < word.length(); index++) {
reverse = word.charAt(index) + reverse;	
}
System.out.println(reverse);
System.out.println(word);

if(reverse == word) {
System.out.print(word + " is a palindrome!");
}
if(reverse != word) {
System.out.print(word + " is not a palindrome!");
}

}
}

File: Hybrid_Task/Javascript/string_palindrome.js
let word = "madam"

let reverse = " "
for(let letter = 0; letter < word.length; letter += 1) {
	 reverse = word[letter] + reverse
}
console.log(reverse)
if(reverse == word) console.log(word + " is a palindrome")
if(reverse != word) console.log(word + " is not a palindrome")